packages/main/src/Table.js

```diff
@@ -428,6 +507,18 @@ class Table extends UI5Element {
 
 		return `${this._id}-showMore-text`;
 	}
+
+	get tableEndDOM() {
+		return this.shadowRoot.querySelector(".ui5-table-end-marker");
+	}
+
+	debounce(fn, delay) {
```

Comment:
IMO move to `base/util/debounde.js`. I think we already use this somewhere? So we could reuse the new file from both places